assets/src/block-validation/icon.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+function IconSVG( { hasBadge } ) {
```

Comment:
Same in this file, I'd suggest using actual `.svg` files

Reply:
I hesitated with some of these because of the dynamic classes, but ended up wrapping them in a span with the dynamic class in cfe19d7fd